### PR TITLE
Fix/Improve plot scripts

### PR DESCRIPTION
**Pull request by glonlas**
## Summary
Since the refactoring in March, the plot scripts have not been updated and some features were broken. 

1. This PR update both plot scripts `plot_dataframe.py` and `plot_profit.py`.
2. In addition, it makes  `plot_dataframe.py` more flexible with Strategy that does not have some indicators previously required by this script. 
3. To make  `plot_dataframe.py`  more useful,  the following indicators have been added (user car select which indicator she/he wants to display): SMA, EMA10/EM50, RSI, Fisher RSI, STOCH Slow and Fast.
4. Remove the hardcoded `backtest-result.json` file in scripts and use the new param `--export-filename` introduced this weekend.
5. Add the support of `--refresh-pairs-cached` param. When the script does not find the Data for the pair, it asked the user to use `--refresh-pairs-cached`, but nothing happened.
6. Add two params `--indicators1` and `--indicators2` to select what indicators to use. These params are dynamics and are compatible with all strategies.

## Quick changelog

- Remove hardcoded `backtest-result.json`file in scripts and use the new param `--export-filename`
- Fix broken features in `plot_dataframe.py` and `plot_profit.py`
- Add the support of `--refresh-pairs-cached` param.
- Improve `plot_dataframe.py` by not crashing if an indicator is not present in the strategy
- Add 6 additional indicator to `plot_dataframe.py`
- Users can select which indicator to display on `plot_dataframe.py` with `--indicators1` and `--indicators2`

## What's new?
`plot_dataframe.py` becomes more flexible and you can select if you want another indicator than MACD. In the examples below it shows the original behavior that display MACD, then another behavior, when you change the configuration to display RSI.

### Display MACD, EMA10 and EMA50
**Generated with params**
```bash
python3 scripts/plot_dataframe.py --pair BTC/EUR --indicators1 ema10,ema50 --indicators2 macd
```
btc-eur-macd

### Display RSI instead of MACD
**Generated with params**
```bash
python3 scripts/plot_dataframe.py --pair BTC/EUR --indicators1 ema10,ema50 --indicators2 rsi
```
btc-eur-rsi

### CODE REVIEW

File: scripts/plot_dataframe.py [hunk 1]
```diff
@@ -35,55 +32,113 @@
...
 
-    tick_interval = analyze.strategy.ticker_interval
+    # Set the ticker to use
+    tick_interval = analyze.get_ticker_interval()
 
+    # Load pqir tickers
     tickers = {}
     if args.live:
         logger.info('Downloading pair.')
         # Init Bittrex to use public API
```
Comment (vertti) [inline scripts/plot_dataframe.py]: Probably should remove the comment about Bittrex here

File: scripts/plot_dataframe.py [hunk 2]
```diff
@@ -35,55 +32,113 @@
...
             'Impossible to load the strategy. Please check the file "user_data/strategies/%s.py"',
             args.strategy
         )
         exit()
 
-    tick_interval = analyze.strategy.ticker_interval
+    # Set the ticker to use
+    tick_interval = analyze.get_ticker_interval()
 
+    # Load pqir tickers
```
Comment (vertti) [inline scripts/plot_dataframe.py]: Typo

File: scripts/plot_dataframe.py [hunk 3]
```diff
@@ -35,55 +32,113 @@
 logger = logging.getLogger(__name__)
 _CONF: Dict[str, Any] = {}
 
+
+# Update the global variable TO_DISPLAY to select which indicator you want to display
+TO_DISPLAY = {
```
Comment (xmatthias) [inline scripts/plot_dataframe.py]: honestly, i don't like the fact that the script needs to be changed to change the displayed indicators.

would something like `--indicators1 sma,ema20,ema40 --indicators2 macd,rsi` specifying the dataframe's non-default columns make sense? (with sensible defaults of course)

Rest looks good to me

File: scripts/plot_dataframe.py (hunk 1 above)
Comment (glonlas) [reply scripts/plot_dataframe.py]: Fixed

File: scripts/plot_dataframe.py (hunk 2 above)
Comment (glonlas) [reply scripts/plot_dataframe.py]: Fixed

File: scripts/plot_dataframe.py (hunk 3 above)
Comment (glonlas) [reply scripts/plot_dataframe.py]: Done. Now you can use `--indicators1 sma,ema20,ema40 --indicators2 macd,rsi`

File: freqtrade/arguments.py [hunk 4]
```diff
@@ -172,7 +174,7 @@ def optimizer_shared_options(parser: argparse.ArgumentParser) -> None:
     @staticmethod
     def hyperopt_options(parser: argparse.ArgumentParser) -> None:
         """
-        Parses given arguments for Hyperopt scripts.
+        Parses given arguments foér Hyperopt scripts.
```
Comment (shusso) [inline freqtrade/arguments.py]: shouldn't it be `for` 😬

Comment (glonlas) [reply freqtrade/arguments.py]: What I did again :disappointed_relieved:
Will be fixed

File: scripts/plot_dataframe.py [hunk 5]
```diff
@@ -35,55 +44,111 @@
...
+
+    plot(fig, filename=os.path.join('user_data', 'freqtrade-plot.html'))
+
+
+def generate_graph(pair, trades, data, args) -> tools.make_subplots:
+    """
+    Generate the graph from the data generated by Backtesting or from DB
+    :param pair: Pair to Display on the graph
+    :param trades: All trades created
+    :param data: Dataframe
```
Comment (xmatthias) [inline scripts/plot_dataframe.py]: `:args ` is missing from the docstring ...

**Pull request merged**